### ISSUE TITLE
new Tree type  is a breaking api change 

### ISSUE THREAD

**Issue by jmatsushita** [purescript-spec/purescript-spec#143]
Hi there,

I believe the change of the `Tree` type in https://github.com/purescript-spec/purescript-spec/pull/139 is a breaking change and would probably benefit from a clearer version bump or a [note in the github release](https://github.com/purescript-spec/purescript-spec/releases/tag/v7.5.1) ?

Cheers,

Jun

**Comment by jmatsushita** [comment]
To clarify, I've been using a fork of https://github.com/purescript-spec/purescript-spec-mocha and `v7.5.1` [breaks here](https://github.com/purescript-spec/purescript-spec-mocha/blob/master/src/Test/Spec/Mocha.purs#L38).

**Comment by fsoikin** [comment]
Hi @jmatsushita, you're absolutely right, it totally slipped my mind that the `Tree` type is part of the public API.

I added a mention of breaking change to the README and the 7.5.1 release description.
I also patched up `purescript-spec-mocha`, it is now up to date.

I do disagree about the version however. Yes, the change is breaking, but it is minor, easily rectified, and in a place that is not normally touched by regular consumers. I feel like a minor version bump is enough.
